tensorflow/lite/kernels/BUILD

```diff
@@ -1,5 +1,4 @@
-load("//tensorflow/lite:build_def.bzl", "tflite_copts")
-load("//tensorflow/lite:build_def.bzl", "tflite_linkopts")
+load("//tensorflow/lite:build_def.bzl", "tflite_copts", "tflite_linkopts")
```

Comment:
oops, I've just realized that we don't use tflite_linkopts() now. could you revert the change?